.github/workflows/linter.yml

```diff
@@ -5,10 +5,7 @@ name: Lint Code Base
 # https://help.github.com/en/articles/workflow-syntax-for-github-actions
 #
 on:
-  push:
-    branches-ignore: [master]
-  pull_request:
-    branches: [master]
+  workflow_dispatch:
```

Comment:
Why changing the triggers for this action?

Reply:
It is annoying to have linter on every push/pr since it displays too many recomendations for the code

Reply:
Linters should run always. This annoyance should be motivation to fix the warnings